src/FormTextarea.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+
+const FormTextarea = (props) => {
```

Comment:
You could save having to name the function by just doing `export default (props) => {...` and dropping `export default FormTextarea;` below.

More of a style thing - up to you.